File: crates/deslop/tests/fixtures/fused-golden-typescript/shape_alpha.ts
export class InventoryDescriptor {
  readonly channel = "alpha-inventory";
  readonly retries = 3;
  readonly endpoint = "https://alpha.example.com/inventory";

  compose(tenant: string): string {
    return this.endpoint + "/" + tenant + "/" + this.channel + "?retries=" + String(this.retries);
  }
}

File: crates/deslop/tests/fixtures/fused-golden-csharp/shape_alpha.cs
namespace Golden.Shapes;

public sealed class InventoryDescriptor
{
    public string Channel { get; } = "alpha-inventory";
    public int Retries { get; } = 3;
    public string Endpoint { get; } = "https://alpha.example.com/inventory";

    public string Compose(string tenant)
    {
        return Endpoint + "/" + tenant + "/" + Channel + "?retries=" + Retries;
    }
}
